plugins/svg/src/SvgFeatureRenderer/components/SvgFeatureRendering.test.js

```diff
@@ -382,6 +384,481 @@ test('processed transcript', () => {
   expect(container.firstChild).toMatchSnapshot()
 })
 
+test('processed transcript (exons + impliedUTR)', () => {
+  const { container } = render(
+    <Rendering
+      width={500}
+      height={500}
+      regions={[{ refName: 'zonk', start: 0, end: 1000 }]}
+      layout={new GranularRectLayout({ pitchX: 1, pitchY: 1 })}
+      features={
+        new Map([
+          [
+            'one',
+
+            // from BCL10 http://localhost:3001/?config=test_data%2Fconfig_demo.json&session=share-stBb1dYlIN&password=rwNsf
+            new SimpleFeature({
+              source: 'BestRefSeq',
+              type: 'gene',
+              start: 85731458,
+              end: 85742315,
+              strand: -1,
+              phase: 0,
+              refName: 'NC_000001.10',
+              id: 'gene1109',
+              dbxref: ['GeneID:8915', 'HGNC:HGNC:989', 'MIM:603517'],
+              name: 'BCL10',
+              description: 'BCL10 immune signaling adaptor',
+              gbkey: 'Gene',
+              gene: 'BCL10',
+              gene_biotype: 'protein_coding',
+              gene_synonym: [
+                'c-E10',
+                'CARMEN',
+                'CIPER',
+                'CLAP',
+                'IMD37',
+                'mE10',
+              ],
+              subfeatures: [
+                {
+                  source: 'BestRefSeq',
+                  type: 'mRNA',
+                  start: 85731458,
+                  end: 85742315,
+                  strand: -1,
+                  phase: 0,
+                  refName: 'NC_000001.10',
+                  id: 'mRNA2437',
+                  parent: 'gene1109',
+                  dbxref: [
+                    'GeneID:8915',
+                    'Genbank:NM_003921.5',
+                    'HGNC:HGNC:989',
+                    'MIM:603517',
+                  ],
+                  name: 'NM_003921.5',
+                  gbkey: 'mRNA',
+                  gene: 'BCL10',
+                  product:
+                    'BCL10 immune signaling adaptor, transcript variant 1',
+                  tag: 'RefSeq Select',
+                  transcript_id: 'NM_003921.5',
+                  subfeatures: [
+                    {
+                      start: 85731458,
+                      end: 85733309,
+                      strand: -1,
+                      type: 'three_prime_UTR',
+                      uniqueId:
+                        'type-Gff3TabixAdapter;type-Gff3TabixAdapter;uri-https://s3.amazonaws.com/jbrowse.org/genomes/hg19/nc-offset-294046480-0_three_prime_UTR_0',
+                      parentId:
+                        'type-Gff3TabixAdapter;type-Gff3TabixAdapter;uri-https://s3.amazonaws.com/jbrowse.org/genomes/hg19/nc-offset-294046480-0',
+                    },
```

Comment:
since the UTR is provided here, this doesn't test the implied utr generation, does it?

Reply:
appears to be because the code for makeUTRs is actually mutating the feature selection. I may want to change makeUTRs to be more pure

Reply:
i don't get it; the test case here is giving a UTR to the renderer, so it doesn't have a chance to make its own implied UTR, does it?